scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -0,0 +1,563 @@
+---
+title: "Integrating scRNA-Seq datasets"
+author: The Data Lab for ALSF
+date: 2022
+output:
+  html_notebook: 
+    toc: true
+    toc_depth: 3
+    toc_float: true
+---
+
+## Objectives
+
+This notebook will demonstrate how to:
+
+- Prepare SCE objects for integration
+- Apply integration methods including `fastMNN` and `harmony`
+- Visually explore the results of integration
+- Learn skills in functional programming in the `tidyverse` with `purrr`
+
+---
+
+In this notebook, we'll perform integration of several scRNA-seq datasets from the [Single-cell Pediatric Cancer Atlas (`ScPCA`)](https://scpca.alexslemonade.org/), a database of uniformly-processed pediatric scRNA-seq data built and maintained by the Data Lab.
+This database currently hosts projects from ten different ALSF-funded labs who have generated single-cell pediatric cancer transcriptomic data with the ultimate goal of making this data easily accessible to investigators (like you!).
+
+Specifically, we'll have a look at four libraries from the [`SCPCP000005` project](https://scpca.alexslemonade.org/projects/SCPCP000005) led by the [Dyer](https://www.stjude.org/research/labs/dyer-lab.html) and [Chen](https://www.stjude.org/research/labs/chen-lab-taosheng.html) labs at St. Jude Children's Research Hospital.
+This project performed single-cell sequencing of pediatric solid tumors, including the rhabdomyosarcoma (RMS) samples we'll be looking at here ([Patel _et al._ (2022)](https://doi.org/10.1016/j.devcel.2022.04.003)).
+
+
+
+We'll be integrating these libraries with two different tools, [`fastmNN`](http://www.bioconductor.org/packages/release/bioc/html/batchelor.html) and [`harmony`](https://portals.broadinstitute.org/harmony/). 
+**Still fleshing this out**
+
+## Set Up
+
+```{r setup}
+# Load libraries
+library(magrittr)
+library(ggplot2)
+library(SingleCellExperiment)
+
+# Set the seed for reproducibility
+set.seed(12345)
+```
+
+
+### Functional programming
```

Comment:
I think we might make this header more action-oriented, like "operating on lists". I don't think we really want to get into all of what functional programming really means, as map functions are only part of it.